spec/models/category_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
+  describe 'associations' do
+    it { should belong_to(:user) }
+    it { should have_many(:bills) }
+  end
```

Comment:
- Hey, let's also add tests that show the category has an icon and a name as in the category model.

![image](https://github.com/lukman155/Budget-Rails-Capstone/assets/59397159/03e27756-383f-4980-acce-ea27f5675072)